src/network/networkd-json.c

```diff
@@ -1023,6 +1025,25 @@ static int dhcp_server_append_json(Link *link, JsonVariant **v) {
         return json_append_one(v, "DHCPServer", w);
 }
 
+static int dhcp6_lease_append_json(Link *link, JsonVariant **v) {
+        _cleanup_(json_variant_unrefp) JsonVariant *w = NULL;
+        int r;
+
+        assert(link);
+        assert(v);
+
+        if (!link->dhcp6_lease)
+                return 0;
+
+        r = json_build(&w, JSON_BUILD_OBJECT(
+                                JSON_BUILD_PAIR_FINITE_USEC("T1", link->dhcp6_lease->lifetime_t1),
+                                JSON_BUILD_PAIR_FINITE_USEC("T2", link->dhcp6_lease->lifetime_t2)));
+        if (r < 0)
+                return r;
+
+        return json_append_one(v, "DHCPv6lease", w);
```

Comment:
We have `DHCPServer` key in the above. So, I think it is better to nest one more level,
```
DHCPv6Client -> Lease -> T1, T2
```
then, we can later expose e.g. client state or so under `DHCPv6Client` field.

Reply:
Updated. Now the output would look like below in JSON.

\"DHCPv6Client\":{\"Leases\":{\"T1\":345600000000,\"T2\":552960000000}}